Lib/defconQt/fontInfo.py

```diff
@@ -38,7 +32,7 @@ def __init__(self, font, parent=None):
         mainLayout.addWidget(buttonBox)
         self.setLayout(mainLayout)
 
-        self.setWindowTitle("Font Info – %s %s" % (
+        self.setWindowTitle(self.tr("Font Info") + " – %s %s" % (
             self.font.info.familyName, self.font.info.styleName))
```

Comment:
Can you avoid using + in your changes and use the existing % or .format for this purpose instead?